### PR TITLE
Match finding and elo formula

### PR DESCRIPTION
**Pull request by whitewind664**

### CODE REVIEW

**Review by smikulas — APPROVED** [review]

File: app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+class Matchmaking {
+    companion object {
+        // TODO if time is available, use glicko2 instead of elo
+        // https://lichess.org/page/rating-systems
+        // http://www.glicko.net/glicko/glicko2.pdf
+
+        private val eloChangeConstant = 30
+
+        private fun eloWinProbability(rating1: Double, rating2: Double): Double {
```
Comment (lupusmalus) [inline app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt:13]: Magic numbers all over the place...

File: app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            return 1.0 / (1.0 + 10.0.pow((rating1 - rating2) / 400.0))
```
Comment (lupusmalus) [inline app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt:14]: magic numbers all over the place

**Review by lupusmalus — APPROVED** [review]
LGTM, though you'll want to get rid of the magic numbers

File: app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt (hunk 2 above)
Comment (whitewind664) [reply app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt:14]: I don't think there's anything I can name them, that's the formula I found

**Pull request merged**